conf/defaults.ini

```diff
@@ -183,6 +183,12 @@ cache_mode = private
 # For "sqlite3" only. Enable/disable Write-Ahead Logging, https://sqlite.org/wal.html. Default is false.
 wal = false
 
+# Skips the migration process for the sqlstore.
+skip_migrations = false
```

Comment:
I'm not sure it's a good idea to expose this and make it visible to users. I understand the motivation -- to be able to override it via environment variable, but I think by exposing this option, people may think it's a good idea to enable it and control migrations manually (it isn't).

I think we should fix our environment variables usage, so that it doesn't depend on values being in config file instead. As for our problem that we're solving, we can easily set this in the config file.